aws/dynamodb/dynamo.tf

```diff
@@ -120,11 +120,6 @@ resource "aws_dynamodb_table" "audit_logs" {
     projection_type = "KEYS_ONLY"
   }
 
-  ttl {
```

Comment:
Since this isn't showing a TF plan change, could you try disabling explicitly first to see if that gives the expected change:
```
enabled = false
```

Reply:
Yes I just pushed the update

Reply:
Looks better! I guess we will remove that block in the future.

Reply:
100% - once this goes to prod we can just remove the block.  Funny that Terraform doesn't catch the removal of the block as a disable action.